page-activities.php

```diff
@@ -55,7 +55,7 @@
                             $tags = get_the_tags($activity->ID);
                         ?>
                         <div class="activities__tag-container">
-                            <?php if(sizeof($tags) > 0): ?>
+                            <?php if(is_array($tags) && sizeof($tags) > 0): ?>
```

Comment:
@kycapstick added is array check here, you shouldn't receive the error anymore

Reply:
This looks great. The last thing is that same fix needs to be added to the single-activities page.

Reply:
Array check was in the incorrect order on the single-activities page.  Should be good now

Reply:
Looks great! Good to merge!